framework/src/modules/reward/module.ts

```diff
@@ -111,11 +110,8 @@ export class RewardModule extends BaseModule {
 		};
 
 		const data = codec.encode(rewardMintedDataSchema, rewardMintedData);
-		context.eventQueue.add(
-			this.name,
-			TYPE_ID_REWARD_MINTED,
-			codec.encode(rewardMintedDataSchema, data),
-			[context.header.generatorAddress],
-		);
+		context.eventQueue.add(this.name, this.name, codec.encode(rewardMintedDataSchema, data), [
```

Comment:
`this.name` is used twice here for module and event name

Reply:
Can give this event a constant name like this one 
> https://github.com/LiskHQ/lisk-sdk/blob/fd2ce98f94c134dc933023d10e17714b749328f8/framework/src/state_machine/constants.ts#L21

agree?

Reply:
Yes if there is a constant for this event in the LIP let's use it. Otherwise I think we can create our own.